### DISCUSSION TITLE
Problématiques

### DISCUSSION THREAD

**Discussion by soleilvermeil** [ozan-guven/HUM-450-Project#2] [Ideas]
Problématique

* Y a-t-il une corrélation entre les lieux d'origines et les lieux de résidence ?
* Plus généralement, qu'est-ce qui caractérise une communauté ?
* * Le nombre d'enfants
* * Âge moyen des habitants (attention aux foyers ou ...)
* * Les lieux d'origines
* * Le nombre de pensionnaires
* * Le métier (cf. les hugenots et l'horlogerie)

À faire :

* Nettoyer un peu la base de données

Difficultés possibles :

* Nettoyer la base de données pour comparer convenablement les entrées
* Quantifier convenablement certaines variables, comme la distribution des âges au sein d'un foyer ou encore le niveau d'aisance

**Comment by ozan-guven** [comment]
Bonnes idées !
